src/Test.UnitTests.Sarif.Driver/Sdk/AnalyzeCommandBaseTests.cs

```diff
@@ -836,7 +836,7 @@ public void AnalyzeCommand_EncodedPaths()
         {
             var logger = new MemoryStreamSarifLogger(dataToInsert: OptionallyEmittedData.Hashes);
             var command = new TestMultithreadedAnalyzeCommand();
-            var uri = new Uri("/new folder/0%1%20%.txt", UriKind.Relative);
+            var uri = new Uri("/new%20folder/0%1%20%.txt", UriKind.Relative);
```

Comment:
>20folder [](http://example.com/codeflow?start=36&length=8)

why did you have to change this?

i am concerned that you've changed some subtle behavior in our service usage.

this is a lousy test because it doesn't describe any expectations of the test. why was it authored? what was it protecting against?

Reply:
Restored the original target, which is a relativeUriWithEncodedFileName. (relative Uri will return the original string in the path, so no problem before the change)

Add one more target, which is a absoluteUriWithEncodedFileName.

My understanding of the purpose of this test case is to ensure no exception is throwing before proceeding with Test rule analyzing.